### PR TITLE
Restore old height infos

### PR DESCRIPTION
**Pull request by elmattic**
## Summary of changes

This PR is a follow-up of #4080, which was restoring the `Liftoff` height entry.

Changes introduced in this pull request:

- Restore `Breeze`, `Smoke`, and `Ignition` entries for _Calibnet_
- Restore `Breeze`, `Smoke`, `Ignition`, `ActorsV2`, `Liftoff`, and `Calico` entries for _Devnet_ and _Butterflynet_
- Add unit tests

Those heights are used at different places in the protocol and can have a significant impact, too.

## Reference issue to close (if applicable)

Closes

## Change checklist

- [x] I have performed a self-review of my own code,
- [x] I have made corresponding changes to the documentation,
- [x] I have added tests that prove my fix is effective or that my feature works (if possible),
- [x] I have made sure the [CHANGELOG][1] is up-to-date. All user-facing changes should be reflected in this document.

[1]: https://github.com/ChainSafe/forest/blob/main/CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -228,7 +231,7 @@ decommissioned.
   `Filecoin.StateAccountKey` lotus-compatible RPC API.
 - [#3744](https://github.com/ChainSafe/forest/pull/3744) Implement the
   `Filecoin.StateLookupID` lotus-compatible RPC API.
-- [#3727](https://github.com/ChainSafe/forest/pull/3727) Added glif.io calibnet
+- [#3727](https://github.com/ChainSafe/forest/pull/3727) Added glif.io the c
```
Comment (hanabi1224) [inline CHANGELOG.md:234]: c?

File: CHANGELOG.md [hunk 2]
```diff
@@ -657,7 +660,7 @@ Notable updates:
 ### Fixed
 
 - [#2796](https://github.com/ChainSafe/forest/pull/2796): Fix issue when running
-  Forest on calibnet using a configuration file only.
+  Forest on the c using a configuration file only.
```
Comment (hanabi1224) [inline CHANGELOG.md:663]: c?

**Review by hanabi1224 — APPROVED** [review]

**Review by LesnyRumcajs — CHANGES_REQUESTED** [review]
What about Butterflynet and Devnet? These (and Liftoff) should be marked as mandatory, preferably via static checks or tests, so that `HEIGHT_INFOS` is invalid without them.

**Comment by elmattic** [comment]
@LesnyRumcajs, do you have any idea why Devnet checks failed?

**Comment by LesnyRumcajs** [comment]
> @LesnyRumcajs, do you have any idea why Devnet checks failed?

Yes. I'll send a PR your way soon.

**Comment by LesnyRumcajs** [comment]
@elmattic https://github.com/ChainSafe/forest/pull/4097 Please take a look to see if the change makes sense. If so, feel free to merge it with your branch.

**Comment by elmattic** [comment]
Thanks! @LesnyRumcajs

**Review by LesnyRumcajs — APPROVED** [review]

**Pull request merged**